src/templates/_elements/tableview/container.html

```diff
@@ -1,14 +1,24 @@
+{% set tableName = tableName ?? '' %}
+{% set instructionId = instructionId ?? "sort-instructions-#{random()}" %}
+
 <div class="tableview{% if context == 'index' %} tablepane{% endif %}">
-    <table class="data fullwidth"{% if structureEditable is defined and structureEditable %} data-structure-id="{{ structure.id }}" data-max-levels="{{ structure.maxLevels }}"{% endif %}>
-        <thead>
+    <span class="visually-hidden" data-status-message aria-live="assertive"></span>
+    <table class="data fullwidth"{% if structureEditable is defined and structureEditable %} data-structure-id="{{ structure.id }}" data-max-levels="{{ structure.maxLevels }}"{% endif %} data-name="{{ tableName }}">
+      <caption class="visually-hidden">{{ tableName }}<span id="{{ instructionId }}" data-sort-instructions>, column headers with buttons are sortable</span></caption>
```

Comment:
Do all tables have at least one sortable column? If not, the sort instructions may need to be conditional.

Reply:
Unfortunately it’s not known by the template, but in general, yes they will be.